src/decision_process/examples.md

```diff
@@ -0,0 +1,297 @@
+
+> Hello! @Alan has proposed to merge this. This is a **reversible decision**,
+> which means that it will be affirmed once the "final comment period" of 10
+> days have passed, unless a team member pauses or suspends the decision.
+>
+> | Team member | State |
+> | --- | --- |
+> | @Alan | **merge** |
+> | @Barbara | **hold** |
+> | @Grace | |
+> | @Niklaus | |
+
+Alan is currently busy at work, though, so by by the time that he and Barbara
+get a chance to talk, 11 days have passed. (Alan and Barbara receive a ping
+from rustbot after a week or so.) Once they get a chance to talk, Alan
+fully addresses Barbara's concern, so Barbara posts:
+
+> @rustbot merge
+
+The comment is updated:
+
+> Hello! @Alan has proposed to merge this. This is a **reversible decision**,
+> which means that it will be affirmed once the "final comment period" of 10
+> days have passed, unless a team member pauses or suspends the decision.
+>
+> | Team member | State |
+> | --- | --- |
+> | @Alan | **merge** |
+> | @Barbara | ~~hold~~ **merge** |
+> | @Grace | |
+> | @Niklaus | |
+
+Barbara's previous ~~hold~~ status links to her previous comment setting her
+status to `hold`, and her current **merge** status links to her more recent
+comment setting her status to `merge`.
+
+At this point, all the statuses are either empty, **abstain**, or **merge**,
+and more than 10 days have passed since the FCP started. Therefore, it
+completes immediately.
+
+## Authoring an RFC (illustration of `rustbot restart`)
+
+After some time, the proposal is completed and an RFC is proposed. This is a
+reversible decision. Alan, as the liaison, proposes to merge the RFC with
+`@rustbot merge`, and the decision making process proceeds as above.
+
+This time, Niklaus has a concern:
+
+> @rustbot hold
+>
+> I have not had time to read this yet! Give me a bit of time to write it up.
+
+After 7 days have passed, rustbot writes to her:
+
+> @Niklaus, I see you have placed a hold but 7 days have passed. Are you any
+> closer to reaching a decision? (cc @Alan)
+
+This continues for a week or two while Alan and Niklaus play "email tag". In
+the interim, Barbara decides she agrees with the RFC, so she uses `@rustbot
+merge` as well. The status now looks like this:
+
+> Hello! @Alan has proposed to merge this PR. This is a **reversible
+> decision**, which means that it will be affirmed once the "final comment
+> period" of 10 days have passed, unless a team member pauses or suspends the
+> decision.
+>
+> | Team member | State |
+> | --- | --- |
+> | @Alan | **merge** |
+> | @Barbara | **merge** |
+> | @Grace | |
+> | @Niklaus | **hold** |
+
+Eventually, Alan and Niklaus find a time to discuss, and Alan agrees that
+Niklaus's concerns are valid, so he makes some major edits to the RFC. Given
+that the RFC is completely different, he decides to restart the clock and
+writes:
+
+> @rustbot restart
+
+This clears the state of all team members (preserving the history) and begins
```

Comment:
Hmm. I was for a moment confused by the combination of "clears the state" and "preserves the history." The example I think does demonstrate the outcome. Still, maybe this phrasing would be better:

```suggestion
This strikes out the state of all team members (clearing their entries while preserving the history) and begins
```

Reply:
I rephrased this, as well as the corresponding language in the reference.